backuprestore.adoc

```diff
@@ -319,10 +319,9 @@ delete old backups when they are no longer needed.
 * `zmbackupquery` -- This command lists the information about ongoing and
 completed backups, including labels and dates.
 * `zmrestore` -- This command executes a full or incremental restore to the
-Zimbra mail server. The *zmrestore* command is performed on a server
-that is running.
-* `zmrestoreoffline` -- This command restores the Zimbra mail server when
-the mailboxd process is stopped.
+Zimbra mail server. Should only be performed on a running mailbox server.
+* `zmrestoreoffline` -- This command restores a full or incremental restore
```

Comment:
nit: inconsistent terminology (see zmrestore directly before where we should also replace Zimbra with {product-name})

Change:
> This command restores a full or incremental restore to the Zimbra mail server.

to

> This command executes a full or incremental restore to the {product-name} mailbox server.

or (use _restore_ or _restoral_ ... your call)

> This command executes a full or incremental restoral to the {product-name} mailbox server.

I didn't see this noted (but may have missed it). An incremental restore implies a full restoral, **plus** the requested incremental data. I'm not sure if we need/want to add that additional info in there. This can be somewhat inferred perhaps from other info in the doc, so it may not be necessary here.